src/test/mochitest/browser.ini

```diff
@@ -655,8 +655,8 @@ support-files =
 [browser_dbg-asm.js]
 [browser_dbg-async-stepping.js]
 [browser_dbg-sourcemapped-breakpoint-console.js]
-[browser_dbg-xhr-breakpoints.js]
 skip-if = (os == "win" && ccov) # Bug 1453549
+[browser_dbg-xhr-breakpoints.js]
```

Comment:
So we are skipping it for windows build?

Reply:
No; the `skip-if` conditions apply to the test *above* the `skip-if` text, so in this case line 657

Reply:
ohh, Thanks